airbyte-integrations/connectors/source-convertkit/source_convertkit/schemas/subscribers.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
https://developers.convertkit.com/#subscribers looks like subscribers support pagination. I guess you need to specify the sort field, sort order, and the cursor in the parameter `from`.

Reply:
Ok, so this is a little more difficult than that.
1. `sort_field` only supports `cancelled_at` value. When it is specified, it only returns unsubscribes (which is undesired behaviour)
2. `from` field only supports dates- when the subscriber was added. For the `full_refresh` we want to get all entries, not specifying the date.

Therefore, I implemented `PageIncrement` paginator.